themes/bootstrap3/templates/myresearch/bulk-action-buttons.phtml

```diff
@@ -6,7 +6,7 @@
 <nav class="bulkActionButtons">
   <div class="bulk-checkbox">
     <input type="checkbox" name="selectAll" class="checkbox-select-all" id="myresearchCheckAll">
-    <label for="myresearchCheckAll"><?=$this->transEsc('select_page')?> | <?=$this->transEsc('with_selected')?>:</label>
+    <label for="myresearchCheckAll"><?=$this->transEsc('select_page')?>:</label>
```

Comment:
This part of the label ("| Selection:") is misleading because it refers to the following buttons/checkboxes. It does not refer to the checkbox where the label is. That's why this part of the caption should be removed from the label element.

Reply:
That makes sense, but should we just move it outside of the label tag rather than deleting it entirely?

Reply:
Well, the label consisted of 2 parts:
"Select Page | with selected" (or with tokens: `select_page | with_selected`)
Now the label says:
"Select all entries of the page" (or with tokens: `select_page`)

- the with_selected part is misleading, so we removed that
- also we have adjusted the translation for the part of the label which is still there: select_page = "Select all entries of the page"

Reply:
Well, i think everything of `with_selectd` can be removed because it does not explain the function of the checkbox itself and it refers to the following elements, like I explained above.

I don't like changing "Select all entries of the page" to "Select All on Page" that much. Can be misleading, because you are not able to select all the page with the checkbox, just all entries.

Sure, I can use a new token for the label, no problem.

Reply:
Thanks, @demiankatz  I did remove "with_selected" from the language files.

To disable the buttons if no checkbox is selected sounds good to me. 
With the additional text "You have selected X items" and numbering the results, I am not sure if that would really be helpful. Numbers don't have any connection or information to the record and that's why they do not contain added value. @claas what do you think?

Reply:
Thanks for the language file update. I think you may have meant to tag @ckaz here.